crates/rooch-key/src/keystore.rs

```diff
@@ -47,30 +59,39 @@ pub enum Keystore<K: Ord, V> {
 }
 
 #[enum_dispatch]
-pub trait AccountKeystore<Addr: Copy, PubKey, KeyPair, Sig, TransactionData>: Send + Sync {
+pub trait AccountKeystore<Addr: Copy, PubKey, KeyPair, Sig, TransactionData, PrivKey>:
```

Comment:
This should be part of https://github.com/rooch-network/rooch/pull/788. As discussed in the PR, Maybe we do not need to distinguish the Keystore via the KeyPair type and TransactionData.

Reply:
Will refactor it in #788 then.